app/models/recipe.rb

```diff
@@ -2,4 +2,7 @@ class Recipe < ActiveRecord::Base
 	validates :title, :ingredients, :directions, :presence => true
 	validates_inclusion_of :difficulty, :in => ["Beginner", "Intermediate", "Advanced"]
 	validates :servings, numericality: {only_integer: true, greater_than_or_equal_to: 1}, :allow_nil => true
+
+	belongs_to :user
```

Comment:
A user can input their own recipes.

Reply:
@vhoof That hasn't come up for discussion yet & I assumed that we would want to restrict visitors without a verified account to write a recipe. Otherwise, we could do: ```User has_many :recipes``` but not ```Recipe belongs_to :user```. I'll remove for now